### PR TITLE
Use new key manager interface

### PR DESCRIPTION
**Pull request by sethvargo**
Part of https://github.com/google/exposure-notifications-server/issues/913

:warning This requires https://github.com/google/exposure-notifications-server/pull/921 be merged first.

**Release Note**

```release-note
NONE
```

### CODE REVIEW

**Comment by sethvargo** [comment]
Test failure is expected since the parent code isn't merged yet.

/hold

**Comment by mikehelmick** [comment]
/retest

dependent change merged

File: go.mod [hunk 1]
```diff
@@ -61,3 +61,5 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
 	honnef.co/go/tools v0.0.1-2020.1.5
 )
+
+replace github.com/google/exposure-notifications-server => ../exposure-notifications-server
```
Comment (mikehelmick) [inline go.mod:65]: remove this - replace w/ an un-anchored dep

**Comment by sethvargo** [comment]
@mikehelmick can you PTanotherL

**Pull request merged**